lib/oxttl/src/toolkit/parser.rs

```diff
@@ -43,7 +44,7 @@ impl<S: Into<String>> From<S> for RuleRecognizerError {
         }
     }
 }
-
+#[derive(Clone)]
```

Comment:
I am not sure all these `Clone` derives are useful anymore

Reply:
Removed all clone derives from oxttl and reintroduced the ones needed to compile.

Reply:
Still need to clone TurtleParser since parse_slice consumes it.